src/runtime/runtime.cpp

```
@@ -272,12 +197,51 @@ main(int argc, char* argv[]) {
 
     std::unique_ptr<context_t> context;
 
+    signal::handler_t signal_set;
+    signal_set.set_default_action(SIGPIPE, signal::handler_base_t::default_action_t::IGNORE);
+
+    std::function<void(const std::error_code& ec, int)> cb = [&](const std::error_code& ec, int) {
+        if(ec) {
+            return;
```

Silently return? Don't do this unless it is cycle breaking.